<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QompMainWin</class>
 <widget class="QMainWindow" name="QompMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>597</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>qomp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icons/qomp.png</normaloff>:/icons/icons/qomp.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { 
	color: black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #C1C7F5, stop: 1 #8ab4f6);
}

QompPlaylistView { 
     background-color: #DFE1F5;
	 border-radius: 5px;
}

QToolButton { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f0f0f0, stop: 0.4 #ebebeb, stop: 0.7 #dedede, stop: 1.0 #d9d9d9); 
     border: 2px solid #799fd8; 
     border-color: #799fd8; 
     border-top-left-radius: 5px; 
     border-bottom-left-radius: 5px; 
     border-bottom-right-radius: 5px; 
     border-top-right-radius: 5px; 
     width: 28px; 
     height: 28px; 
	padding: 1px;
	icon-size: 28px;
}

QToolButton:hover { 
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #f0f0f0, stop: 1 #DFE1F5);
	border: 2px solid #8ab4f6;
}

QToolButton:pressed { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #8faed5, stop: 0.4 #6dadfa, stop: 0.7 #368ef9, stop: 1.0 #358bf3); 
     border: 2px solid #698ebf; 
     border-color: #698ebf; 
     border-top-left-radius: 5px; 
     border-bottom-left-radius: 5px; 
     border-bottom-right-radius: 5px; 
     border-top-right-radius: 4px; 
}

QToolButton:checked { 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0.191, y2:1, stop:0 rgba(223, 225, 245, 255), stop:0.39196 rgba(207, 209, 245, 255), stop:0.859296 rgba(109, 160, 219, 255), stop:1 rgba(62, 85, 221, 255));
     border: 2px solid #698ebf; 
     border-color: #698ebf; 
     border-top-left-radius: 5px; 
     border-bottom-left-radius: 5px; 
     border-bottom-right-radius: 5px; 
     border-top-right-radius: 4px; 
}

QSlider::handle:horizontal {
 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #aaaaaa, stop: 0.4 #d5d5d5, stop: 0.7 #d5d5d5, stop: 1.0 #c6c6c6); 
    border: 2px solid #698ebf; 
	border-radius: 5px;
	width: 10px;
}

QLabel { 
    color: #3244C9; 
}

QLabel#lb_artist_title, QLabel#lb_track_title, QLabel#lb_playtime {
	background-color: #DFE1F5;
	border-top-left-radius: 2px;
	border-bottom-left-radius: 2px;
	border-top:  1px solid #698ebf;
	border-bottom:  1px solid #698ebf;
	border-left:  1px solid #698ebf;
}

QLabel#lb_artist, QLabel#lb_title, QLabel#lb_playtime {
	background-color: #DFE1F5;
	border-top-right-radius: 2px;
	border-bottom-right-radius: 2px;
	border-top:  1px solid #698ebf;
	border-bottom: 1px solid #698ebf;
	border-right: 1px solid #698ebf;
}

QMenu {
    background-color: #8ab4f6;
    border: 1px solid #698ebf;
}

QMenu::item {
    background-color: transparent;
}

QMenu::item:selected {
	color: black;
   	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                stop: 0 #DFE1F5, stop: 1 #8ab4f6);
}

QLCDNumber{
	border-radius: 5px;
	border: 2px solid #698ebf;
	color: #3244C9;
}

QToolTip {
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f0f0f0, stop: 0.4 	#ebebeb, stop: 0.7 #dedede, stop: 1.0 #d9d9d9);
	border-radius: 5px; 
	border: 2px solid #698ebf;
	padding: 2px;
	color: black;
}

 QScrollBar:vertical {
     border: 2px solid #799fd8;
     background: #DFE1F5;
     width: 18px;
	 margin: 25px 0px;
 }
 QScrollBar::handle:vertical {
     min-height: 20px;
	 border-radius: 5px;
	 border: 2px solid #799fd8;
	 background: rgba(62, 85, 221, 255);
 }
 QScrollBar::add-line:vertical {
     border: 2px solid #799fd8;
     background: rgba(62, 85, 221, 255);
     height: 20px;
	 border-bottom-left-radius: 5px;
     border-bottom-right-radius: 5px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
	 image: url(:/icons/arrow-down);
	 margin-bottom: 5px;
 }

 QScrollBar::sub-line:vertical {
     border: 2px solid #799fd8;
	 border-top-left-radius: 5px;
     border-top-right-radius: 5px;
     background: rgba(62, 85, 221, 255);
     height: 20px;
	 subcontrol-position: top;
     subcontrol-origin: margin;
	 image: url(:/icons/arrow-up);
	 margin-top: 5px;   
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="lcd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lb_artist_title">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Artist:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lb_artist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lb_track_title">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Title:</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lb_title">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="tb_mute">
        <property name="toolTip">
         <string>Mute</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/volume</normaloff>:/icons/volume</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="seekSlider">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="singleStep">
       <number>1000</number>
      </property>
      <property name="pageStep">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="tb_prev">
        <property name="toolTip">
         <string>Previous</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/prev</normaloff>:/icons/prev</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_play">
        <property name="toolTip">
         <string>Play/Pause</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/play</normaloff>:/icons/play</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_stop">
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/stop</normaloff>:/icons/stop</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_next">
        <property name="toolTip">
         <string>Next</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/next</normaloff>:/icons/next</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="tb_repeatAll">
        <property name="toolTip">
         <string>Repeat All</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/repeat</normaloff>:/icons/repeat</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QompPlaylistView" name="playList">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QToolButton" name="tb_open">
        <property name="toolTip">
         <string>Add</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_load">
        <property name="toolTip">
         <string>Open Playlist</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/load</normaloff>:/icons/load</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_save">
        <property name="toolTip">
         <string>Save Playlist</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_clear">
        <property name="toolTip">
         <string>Clear</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/delete</normaloff>:/icons/delete</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_playtime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>74</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>74</width>
          <height>18</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QompBusyLabel" name="lb_busy">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbOptions">
        <property name="toolTip">
         <string>Settings</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/options</normaloff>:/icons/options</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QompBusyLabel</class>
   <extends>QLabel</extends>
   <header>qompbusylabel.h</header>
  </customwidget>
  <customwidget>
   <class>QompPlaylistView</class>
   <extends>QListView</extends>
   <header>qompplaylistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
